### DISCUSSION TITLE
小米手環怎麼分析睡眠

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#736] [General]
雖然我沒有小米手環，但還是很好奇小米手環到底是怎麼分析使用者的睡眠時間跟品質的
小小一個手環竟然可以做那麼多的事，十分有趣。
https://www.doyouknow.wiki/a/202107/618171.html

**Comment by HsuChen88** [comment]
連結不能使用呢，不過雖然我沒有Apple Watch，但我爸有，可以問他。

**Reply by wu51000** [reply]
哈哈哈那你甚至可以跟他借來用 好羨慕嗚嗚
我按可以捏！我再貼一次給你～～
https://www.doyouknow.wiki/a/202107/618171.html

**Reply by HsuChen88** [reply]
連結BAD

**Comment by pigeon9975** [comment]
我有用過小米手環和紅米手錶兩種類型，我覺得小米手錶在測量心律、紀錄睡眠，還有他的手機app都比紅米手錶好用很多！

**Comment by yjpxi** [comment]
以我使用的經歷，早上起床都會看報告查看我睡眠是屬於淺度睡眠還是深度睡眠，小米手環會將我的心率數據和運動數據輸入到其內建的演算法中，演算法能夠區分你睡眠中的不同階段，例如淺度睡眠、深度睡眠和快速眼動睡眠，也會有一份詳細的睡眠報告，包括總睡眠時間、各個階段的時間分佈以及睡眠質量評分，我是覺得蠻準確的。
